packages/liveblocks-client/src/index.ts

```diff
@@ -18,7 +13,14 @@ export type {
   Room,
   StorageUpdate,
   User,
-} from "./types";
+} from "@liveblocks/core";
+export {
+  createClient,
```

Comment:
should `createClient` be part of `@liveblocks/client` instead of being imported from `core`?

Reply:
Makes sense, thanks 🙌